packages/cosmic-swingset/test/prometheus.test.ts

```diff
@@ -73,7 +75,7 @@ test('Prometheus metric definitions', async t => {
       const url = `http://localhost:${port}/metrics`;
       const text = await fetch(url).then(resp => resp.text());
       const metaLines = text.match(/^#.*/gm) || [];
-      const keys = [];
+      const keys = [] as string[];
```

Comment:
Some files do `const $name: $type[] = [];` while others do `const $name = [] as $type[];`. Can we settle on a preferred pattern?

Reply:
I find the latter more readable but I'm fine allowing both.

Reply:
FWIW, I also find `[] as $type` to be more readable.